test/fixtures/block-element-grouping.html

```diff
@@ -0,0 +1,85 @@
+<html>
+  <head>
+    <title>Sites Franklin Example</title>
+    <link rel="canonical" href="https://main--xwalk-example--buuhuu.hlx.page/">
+    <meta name="description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
+    <meta property="og:title" content="Sites Franklin Example">
+    <meta property="og:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
+    <meta property="og:url" content="https://main--xwalk-example--buuhuu.hlx.page/">
+    <meta property="og:image" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
+    <meta property="og:image:secure_url" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:title" content="Sites Franklin Example">
+    <meta name="twitter:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
+    <meta name="twitter:image" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
+    <meta name="foobar" content="hello world">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="/scripts/lib-franklin.js" type="module"></script>
+    <script src="/scripts/scripts.js" type="module"></script>
+    <link rel="stylesheet" href="/styles/styles.css">
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div>
+        <div class="teaser-grouping">
+          <div>
+            <div>
+              <picture>
+                <img src="/content/dam/teaser-background.png" alt="A group of people sitting on a stage">
+              </picture>
+            </div>
+          </div>
+          <div>
+            <div>
+              <p>Adobe Experience Cloud</p>
+              <h2>Meet the Experts</h2>
+              <p>Join us in this ask me everything session ...</p>
+              <p><a href="https://link.to/more-details">More Details</a></p>
+              <p><strong><a href="https://link.to/sign-up">RSVP</a></strong></p>
+            </div>
+          </div>
+        </div>
+        <div class="teaser-grouping">
+          <div>
+            <div>
+              <picture>
+                <img src="/content/dam/teaser-background.png" alt="A group of people sitting on a stage">
+              </picture>
+            </div>
+          </div>
+          <div>
+            <div>
+              <h2>Meet the Experts</h2>
```

Comment:
Can you add a test to check what happens if there is no `<hX>`? 

I would assume the first paragraph becomes the subtitle, any other paragraph becomes part of the description until we match the first CTA.

Reply:
added in https://github.com/mhaack/helix-importer/pull/3